lib/shared/store.js

```diff
@@ -215,7 +215,7 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
 
       if (_.some([
         (!_.isNumber(action.data.speed) || !_.isFinite(action.data.speed)) && action.data.speed !== null,
-        action.data.speed === undefined
+        _.isUndefined(action.data.speed)
```

Comment:
You should be able to just do `_.isNil(action.data.speed)` instead of checking against null & undefined here